modules/core/src/test/java/org/apache/synapse/config/xml/eventing/EventPublisherMediatorSerializerTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.apache.synapse.config.xml.eventing;
+
+import org.apache.axiom.om.OMElement;
+import org.apache.synapse.Mediator;
+import org.apache.synapse.SynapseException;
+import org.apache.synapse.mediators.builtin.CallMediator;
+import org.apache.synapse.mediators.eventing.EventPublisherMediator;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Test class for EventPublisherMediatorSerializer
+ */
+public class EventPublisherMediatorSerializerTest {
+
+    /**
+     * Passing unsupported mediator.
+     */
+    @Test(expected = SynapseException.class)
+    public void testCreateEventSource() {
+            EventPublisherMediatorSerializer mediatorSerializer = new EventPublisherMediatorSerializer();
```

Comment:
Shall we update the code alignment?

Reply:
Fixed in https://github.com/wso2/wso2-synapse/pull/1050/files

Reply:
Foxed in https://github.com/wso2/wso2-synapse/pull/1051